Description: Use the installed files for osdmenu rather than the ones in the
 current dir.

--- a/share/osdmenu/default.cfg
+++ b/share/osdmenu/default.cfg
@@ -1,4 +1,4 @@
-dir share/osdmenu/default
+dir /usr/share/vlc/osdmenu/default
 action	key-play-pause (0,0)
 	unselect unselected.png
 	select   selection/play_pause.png
--- a/modules/video_filter/osdmenu.c
+++ b/modules/video_filter/osdmenu.c
@@ -105,7 +105,7 @@
 #if defined( WIN32 ) || defined( UNDER_CE )
 #define OSD_DEFAULT_CFG "osdmenu/default.cfg"
 #else
-#define OSD_DEFAULT_CFG "share/osdmenu/default.cfg"
+#define OSD_DEFAULT_CFG "/usr/share/vlc/osdmenu/default.cfg"
 #endif
 
 #define OSD_UPDATE_MIN     0
